### PR TITLE
Update quick install script

### PR DESCRIPTION
**Pull request by johnugeorge**
Update quick install script with latest version of Istio and Knative

### CODE REVIEW

File: hack/quick_install.sh [hunk 1]
```diff
@@ -17,11 +17,11 @@ Help() {
    echo
 }
 
-export ISTIO_VERSION=1.20.4
-export KNATIVE_OPERATOR_VERSION=v1.14.5
-export KNATIVE_SERVING_VERSION=1.13.1
+export ISTIO_VERSION=1.23.2
```
Comment (spolti) [inline hack/quick_install.sh:20]: We are actually on Istion v1.23.0, as it is a minor version update, it should be fine.

**Review by spolti — APPROVED** [review]
just a minor comment.

/lgtm

File: hack/quick_install.sh [hunk 2]
```diff
@@ -17,11 +17,11 @@ Help() {
...
+export KNATIVE_OPERATOR_VERSION=v1.15.7
```
Comment (Jooho) [inline hack/quick_install.sh:21]: KNATIVE 1.16 is planned to release next week. why don't we wait 1 week and use that?

**Pull request merged**